php/Util.php

```diff
@@ -205,19 +205,19 @@ public function get_post_type_slug() {
 	 * Get a relative URL from a path.
 	 *
 	 * @param string $path The absolute path to a file.
-	 *
 	 * @return string
 	 */
 	public function get_url_from_path( $path ) {
+		if ( empty( $path ) ) {
+			return $path;
+		}
 		$abspath = ABSPATH;
 
 		// Workaround for weird hosting situations.
 		if ( trailingslashit( ABSPATH ) . 'wp-content' !== WP_CONTENT_DIR && isset( $_SERVER['DOCUMENT_ROOT'] ) ) {
 			$abspath = sanitize_text_field( wp_unslash( $_SERVER['DOCUMENT_ROOT'] ) );
 		}
 
-		$stylesheet_url = str_replace( untrailingslashit( $abspath ), '', $path );
```

Comment:
This was a PHP `8` error in PHPUnit tests. 

If `$path` is `null`, `str_replace()` can't accept `null` as the 3rd argument.

Reply:
So on line 212, this exits if `$path` is empty.